### PR TITLE
API: Stake | get stake

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
image

File: src/test/java/tests/tokens/Token.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    @Test(description = "verify that get a token with tokenId valid | success", groups = {"token"})
+    public void getATokenSuccess(){
+        tokenSteps.getAToken("asset1ee0u29k4xwauf0r7w8g30klgraxw0y4rz2t7xs")
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+    }
+    @Test(description = "verify that get a token with tokenId invalid | fail", groups = {"token"}, dataProvider = "tokenIdInvalid")
+    public void getATokenFail(String tokenId){
+        tokenSteps.getAToken(tokenId)
+                .then_verifyErrorResponse(HttpURLConnection.HTTP_BAD_REQUEST,"Token not found", "BC_404-TOKEN_NOT_FOUND");
```
Comment (Sotatek-DavidNguyen4) [inline src/test/java/tests/tokens/Token.java:21]: don't should put message like this

File: src/test/java/tests/tokens/TokenTxs.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import util.CreateParameters;
+
+import java.net.HttpURLConnection;
+import java.util.Map;
+
+public class TokenTxs extends BaseTest {
+    private TokenSteps tokenSteps = new TokenSteps();
+    @Test(description = "get token txs valid param", groups = {"token"})
+    public void getTokenTxs(){
+        tokenSteps.getTokenTxs("asset1d9v7aptfvpx7we2la8f25kwprkj2ma5rp6uwzv")
```
Comment (Sotatek-DavidNguyen4) [inline src/test/java/tests/tokens/TokenTxs.java:16]: put this above test

File: src/test/java/tests/tokens/TokenTxs.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+    }
+    @Test(description = "get token txs invalid param", groups = {"token"}, dataProvider = "paramTokenTxsParamInvalid")
+    public void getTokenTxsInvalidParam(Object page, Object size){
+        Map<String, Object> param = new CreateParameters()
+                .withPage(page)
+                .withPageSize(size)
+                .getParamsMap();
+        tokenSteps.getTokenTxsParamInvalid("asset1d9v7aptfvpx7we2la8f25kwprkj2ma5rp6uwzv", param)
```
Comment (Sotatek-DavidNguyen4) [inline src/test/java/tests/tokens/TokenTxs.java:25]: here too

**Review by Sotatek-DavidNguyen4 — APPROVED** [review]

**Pull request merged**